internal/cli/adv2new/adv2new.go

```diff
@@ -0,0 +1,25 @@
+package adv2new
+
+import (
+	"github.com/mongodb-labs/atlas-cli-plugin-terraform/internal/cli"
+	"github.com/mongodb-labs/atlas-cli-plugin-terraform/internal/convert"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	o := &cli.BaseOpts{
+		Fs:      afero.NewOsFs(),
+		Convert: convert.AdvancedClusterToNew,
+	}
+	cmd := &cobra.Command{
+		Use:   "advancedClusterToNew",
```

Comment:
```suggestion
		Use:   "advancedClusterV2",
```

just for sanity, how would the full command look like?

Reply:
some equivalent examples to use it with names or aliases are:
```
atlas tf adv2new -f example.in.tf -o example.out.tf
atlas terraform advancedClusterToNew --file example.in.tf --output example.out.tf

# current cluster to adv_cluster command
atlas tf clu2adv -f example.in.tf -o example.out.tf
atlas terraform clusterToAdvancedCluster --file example.in.tf --output example.out.tf
```

you can also mix names and aliases in the same command

I like the convert command pattern name of `xxxToyyy` and alias `x2y`, maybe `advancedClusterToV2` and `adv2v2` ?  (not convinced with so many 2, that's why i tried to avoid 2 in the command name).

Having this context,  what command and alias do you think it's better?

Reply:
that's fair, changed here: https://github.com/mongodb-labs/atlas-cli-plugin-terraform/pull/58/commits/32398acbcd8299e16c4f6fd03cd459596532382a